### PR TITLE
feat: introduce lightbox on Media page

### PR DESCRIPTION
**Pull request by ryuk156**
This PR introduces the Lightbox on the Media page,  improves the quality of images, and also provides a pagination feature to the Media page

Screenshots:
image

image

### CODE REVIEW

File: src/layout/css/_media.scss [hunk 1]
```diff
@@ -14,6 +14,8 @@
   top: 0;
   width: 100vw;
   height: 100vh;
+  -webkit-backdrop-filter: blur(10px);
+  backdrop-filter: blur(10px);
   background-color: #292929;
   opacity: 1;
```
Comment (majordwarf) [inline src/layout/css/_media.scss]: Set `opacity` to 0.5

File: src/components/Media/Media.jsx [hunk 2]
```diff
@@ -31,7 +30,6 @@ const Gallery = () => {
 
   const [imageDisplay, setImageDisplay] = useState(false);
   const [imageToShow, setImageToShow] = useState(``);
-  const [blur, setblur] = useState(``);
```
Comment (majordwarf) [inline src/components/Media/Media.jsx]: You removed the hook but forgot to remove associated event handler `setblur`.

File: src/layout/css/_media.scss (hunk 1 above)
Comment (ryuk156) [reply src/layout/css/_media.scss]: done

File: src/components/Media/Media.jsx (hunk 2 above)
Comment (ryuk156) [reply src/components/Media/Media.jsx]: done

**Review by majordwarf — APPROVED** [review]

**Pull request merged**